lib/route.js

```diff
@@ -142,7 +144,9 @@ function buildRouting (options) {
           case 'both':
           default:
             afterRouteAdded.call(this, '', notHandledErr, done)
-            afterRouteAdded.call(this, path, notHandledErr, done)
+            if (ignoreTrailingSlash !== true) {
```

Comment:
Would you mind adding a quick comment explaining this?

Reply:
Sure! Something like "If ignoreTrailingSlash is set to true we need to add just the '' route to prevent adding an incomplete one."  is OK?

Reply:
that works, yes!